contents/tutorials/delayed-survey.md

```diff
@@ -0,0 +1,66 @@
+---
+title: How to show a survey after a delay
+date: 2023-11-01
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+tags: ['surveys']
+---
+
+To get the most accurate responses for your surveys, you want users to experience your site and product before asking. You want to wait and let them have their first impressions, instead of showing a survey right away.
+
+To help you do this, you can delay your surveys and this tutorial shows you how. It goes over how to create a survey in PostHog and then implement a delay in your app for that survey.
+
+## Create your delayed survey
+
+First, we create our survey in PostHog by going to the [Surveys tab](https://app.posthog.com/surveys) and clicking "New survey." Choose any template or create a blank survey, fill out the details, and open the "Targeting" section. 
+
+In the targeting section, switch from "All users" to "Users who match…". Add `.delayed-survey` under "Selector matches" (the **leading period** is important) and click "Save as draft." Our implementation will add this class to the page when want the survey to show.
```

Comment:
I think we're missing a paragraph describing what we'll do first. e.g. to create a survey with a delay, we'll target pages with a css class and then..."

This will make it easier to follow what we're doing

Reply:
Rewrote to try to make this clearer